src/main/java/com/google/api/codegen/GeneratorMain.java

```diff
@@ -149,6 +149,17 @@
           .argName("GRPC-SERVICE-CONFIG")
           .required(false)
           .build();
+  private static final Option TRANSPORT =
+      Option.builder()
+          .longOpt("transport")
+          .desc(
+              "List of transports to support. Valid transport names ('grpc' or 'rest') are"
+                  + " separated by '+'. Default is 'grpc'. NOTE: for now, GAPICs support only"
```

Comment:
This comment as phrased makes sense with the code changes I suggested in the last pass, where (a) the default list is `grpc+rest` and (b) because we're not yet implementing dual transport, we then look at only the first element in the list as per my #[1] comment: `all_transports = all_transports[0:1]`

Reply:
Please update the comment if you're not handling the list option: you're not handling the "+" separated list unless you make changes like the ones I suggested,.